cms/static/js/views/content_group_details.js

```diff
@@ -21,10 +31,65 @@ define([
 
         initialize: function() {
             this.template = this.loadTemplate('content-group-details');
+            this.listenTo(this.model, 'change', this.render);
         },
 
         render: function() {
-            this.$el.html(this.template(this.model.toJSON()));
+           var attrs = $.extend({}, this.model.attributes, {
+                usageCountMessage: this.getUsageCountTitle(),
+                outlineAnchorMessage: this.getOutlineAnchorMessage(),
+                index: this.model.collection.indexOf(this.model)
+            });
+            this.$el.html(this.template(attrs));
+            return this;
+        },
+
+        showContentGroupUsages: function(event) {
+            if (event && event.preventDefault) { event.preventDefault(); }
+            this.model.set('showContentGroupUsages', true);
```

Comment:
Why is this in the model? It doesn't seem right to me to modify the model to hide and show usages. This means that any views listening to changes will refresh unnecessarily. I'd rather the model be a pure data model, and that UI state be captured in the UI somewhere else (either as classes added and removed to/from elements, or as fields of the view).